pyproject.toml

```diff
@@ -1,2 +1,20 @@
-[tool.black]
```

Comment:
We still need this. That is why the black check was failing. It was working with the default line length of 80.